os/include/tinyara/fs/ioctl.h

```diff
@@ -161,10 +161,9 @@
 										 * OUT: Bytes writable to this fd
 										 */
 
-#define FIONBIO         _FIOC(0x000b)     /* IN:  Boolean option takes an
-                                           *      int value.
-                                           * OUT: Origin option.
-                                           */
+#define FIONBIO         _FIOC(0x000b)     /* IN:  Boolean option takes an int value.
+										 * OUT: Origin option.
+										 */
```

Comment:
why these changes needed

Reply:
because of coding rules violation error

Reply:
same as previous
but for this case, if you use only spaces in this line, then no violation for indentation.
I prefer spaces because of current view. (Github shows not aligned indendation with tabs)

Reply:
yes. but it gave me coding rules violation error while i hadn't touched these files.
so to fix these errors. i made these changes